src/components/editor/faultTree/shapes/RenderTree.tsx

```diff
@@ -28,6 +40,9 @@ const renderTree = (container, node, parentShape = null) => {
   if (has(node, "probabilityRequirement")) {
     nodeShape.attr(["probabilityRequirementLabel", "text"], node.probability.toExponential(2));
   }
+  if (shouldHighlight) {
+    nodeShape.attr("body/stroke", "red");
```

Comment:
I cannot test it so I cannot see, but shouldn't it be orange?

Reply:
Mainly wanted to do/test logic here. Design part i'll change in next PR if its ok.

Reply:
Changed to orange.